src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
```

Comment:
According to BEM and HTML formatting you should change the class names and format the HTML as shown below

```suggestion
      <nav class="navigation">
        <ul class="navigation__list">
          <li class="navigation__item">
            <a href="#" class="navigation__link is-active">
              Apple
            </a>
          </li>
```

[BEM](https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem)

[HTML formatting](https://mate-academy.github.io/style-guides/htmlcss.html#html-formatting)